src/cameras/ArrayCamera.js

```diff
@@ -31,6 +31,8 @@ class ArrayCamera extends PerspectiveCamera {
 		 */
 		this.isArrayCamera = true;
 
+		this.isMultiViewCamera = false;
```

Comment:
I could get rid of this and just check if xr is using multiview.

Reply:
It seems the code in `nodes/accessors/Camera.js`is easier to read if we can evaluate the flag right from the array camera.